### ISSUE TITLE
Why does the font change automatically at runtime?

### ISSUE THREAD

**Issue by DongCher** [ComponentFactory/Krypton#204]
I am Developer in South Korea.
I am testing Krypton5480 and GrapeCity.Spread.Net. (ver 11.45.20183.0)

==> vs2015, Win10 x64, C#, Winform, KryptonToolKit5480, .Net FrameWork 4.8
        GrapeCity.SpreadNet

Why does the font change automatically at runtime?

[KryptonSuiteTest.zip]

==> Design Time
![DesignTime_Krypton](https://user-images.githubusercontent.com/31262463/78738767-a2fc1300-798d-11ea-9bf2-7e1555c0ac9a.PNG)

==> Runtime Normal
![Runtime_Normal](https://user-images.githubusercontent.com/31262463/78738788-ac857b00-798d-11ea-979d-cbcd91ee44b2.PNG)

==> Runtime Krypton
![Runtime_Krypton](https://user-images.githubusercontent.com/31262463/78738803-b5764c80-798d-11ea-83fa-fb92ec0ca82c.PNG)

**Comment by DongCher** [comment]
![DesignTimeFont](https://user-images.githubusercontent.com/31262463/78738439-e43ff300-798c-11ea-92a9-f1e3de0b7d00.PNG)
![RuntimeFont](https://user-images.githubusercontent.com/31262463/78738441-e4d88980-798c-11ea-9f30-3cf2eef7b9c0.PNG)

**Comment by Wagnerp** [comment]
Hi @DongCher 

I don't think this is the right place to report 5.480 issues. Please refer to [here](https://github.com/Krypton-Suite/Standard-Toolkit) for support.

Thanks!

**Comment by uggaldeniya** [comment]
did you find a solution for this ?

**Comment by Wagnerp** [comment]
> did you find a solution for this ?

@uggaldeniya It might be the font/language pack. Does this still occur on 6.2111?

**Comment by uggaldeniya** [comment]
@Wagnerp  Yes. I tried that version too. it seems like font overrides by san serif.
![Captsure](https://user-images.githubusercontent.com/11691372/143770516-9c68820f-a240-47bd-a7cc-8b7aa6e2521c.PNG)
